### PR TITLE
Updated for #1354.

### PR DESCRIPTION
**Pull request by yoshimis**

### CODE REVIEW

**Comment by yoshimis** [comment]
@randriova I have fixed issues you reported. For VIEW ON STACKBLITZ, I will create another PR to cover whole topics in JP folder.

File: jp/components/toc.yml [hunk 1]
```diff
@@ -19,32 +19,32 @@
     - name: プロジェクト テンプレートの認証
       href: general/cli/auth-template.md
       new: false
-- name: WPF to Angular guide
+- name: WPF vs Angular 比較ガイド
```
Comment (valadzhov) [inline jp/components/toc.yml]: @yoshimis All the topics bellow are in English. We should hide them if they won't be localized before the next JP push to production.

**Comment by valadzhov** [comment]
@yoshimis there are merge conflicts in three files (listed above), could you please resolve them because otherwise the branch can't be merged.

**Comment by yoshimis** [comment]
@valadzhov Conflicts resolved.

**Pull request merged**